### DISCUSSION TITLE
Import error from CC3

### DISCUSSION THREAD

**Discussion by mimimi999** [soupday/cc3_blender_tools#83] [Q&A]
![1](https://user-images.githubusercontent.com/107269708/173112416-b0014b4c-ac7a-4897-bb90-5edbd9f27603.png)

I used the latest version of the add-on because it was renamed from the previous add-on.
When I export the character from cc3 with FBX and import it into the blender, the material of the eyes turns black.

When I was changing the material settings from the N panel of the add-on, I noticed that the setting values ​​of the two materials were strange.

I can improve it by fixing it every time I import it, but am I the only one who has this symptom?

**Comment by mimimi999** [comment]
By the way, this looks only when it's a cycle render.

**Comment by soupday** [comment]
The blackness in the eyes is caused by the Random Walk subsurface scattering in Cycles X.

If you're rendering in Cycles: Go to **Character Build Settings** panel > set the render target from Eevee to Cycles. Rebuild the materials. This adjusts all the SSS settings to work better with Cycles. It also makes any hair materials use the correct cycles anisotropic highlights.

You could also fix yourself it by setting the subsurface scattering method in the affected materials to Christensen-Burley and reducing the subsurface values.
